Gordon360/Authorization/StateYourBusiness.cs

```diff
@@ -686,7 +715,30 @@ private async Task<bool> CanUpdateAsync(string resource)
 
                         return false;
                     }
+                    
+                case Resource.RECIM_ACTIVITY:
+                case Resource.RECIM_SERIES:
+                case Resource.RECIM_SPORT:
+                    {
+                        return _participantService.IsAdmin(user_name);
+                    }
 
+                case Resource.RECIM_TEAM:
+                    {
+                        var teamID = (int)context.ActionArguments["teamID"];
+                        return _teamService.IsTeamCaptain(user_name, teamID) || _participantService.IsAdmin(user_name);
```

Comment:
Remember, we have a specific way of doing this that does type checking rather than just casting.
```suggestion
                        return (context.ActionArguments["teamID"] is int teamID
                            && _teamService.IsTeamCaptain(user_name, teamID))
                            || _participantService.IsAdmin(user_name);
```

Reply:
I actually recommended in my review that we use the `TryGetValue` pattern because it avoids the unsafe dictionary access that both of the other patterns use. That is, when you call `context.ActionArguments["someString"]`, if `"someString"` is not a key in the dictionary, it throws an error, which is not desirable in this context.